docs/curl.1

```diff
@@ -150,6 +150,10 @@ version. (Added in 7.33.0)
 .IP "--http2"
 (HTTP) Tells curl to issue its requests using HTTP 2. This requires that the
 underlying libcurl was built to support it. (Added in 7.33.0)
+.IP "--http2-no-upgrade"
+(HTTP) Tells curl to issue its requests using HTTP 2 over clean TCP without 
+HTTP/1.1 Upgrade. This requires that the underlying libcurl was built to 
+support it. (Added in 7.49.0)
```

Comment:
So what happens if this option is set and a HTTPS URL is given? I think the docs should say...